### PR TITLE
PR: Update Changelog, Announcements and metadata files for 6.0.2

### LINKED ISSUE #22751: Spyder 6.0.2 release
This is an issue to track comments, questions, pending things, and general changes regarding dependencies done for the Spyder 6.0.2 release.

New/updated dependencies:
* [x] spyder-kernels: 3.0.1 - pending release (https://github.com/spyder-ide/spyder-kernels/milestone/75)
* [x] QtConsole: 5.6.1 - pending release (https://github.com/jupyter/qtconsole/milestone/51) 

RC releases:
* [x] 6.0.2rc1 (https://github.com/spyder-ide/spyder/issues/22685)

Other things:
* [x] Translations - https://github.com/spyder-ide/spyder/pull/22753
* [x] Update `RELEASE` instructions related with RC releases (see #22757 )

### PR DESCRIPTION
**Pull request by dalthviz**
### Issue(s) Resolved

Fixes #22751

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: dalthviz

### CODE REVIEW

File: changelogs/Spyder-6.md [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+## Version 6.0.2 (2024/10/31)
+
+### Important fixes
+
+* Several fixes and improvements to the update detection mechanism.
+* Fix SSH tunneling info handling for remote kernels connection and add remote client tests.
+* Fix issue related with Matplotlib inline backend registration.
+* Handle kernel fault file not being available.
+* Fix Debugger toolbar buttons and restore missing debugging buttons.
+* Update QtConsole constraint to 5.6.1 to support move cursor ANSI codes.
```
Comment (ccordoba12) [inline changelogs/Spyder-6.md:12]:
```suggestion
* Fix plots not being generated with the Matplotlib inline backend.
* Restore missing debugger buttons to the main toolbar.
* Several fixes and improvements to the update detection mechanism.
* Fix SSH tunneling info handling for remote kernels connection and add remote client tests.
* Handle kernel fault file not being available.
* Update QtConsole constraint to 5.6.1 to support ANSI codes that move the cursor.
```

File: Announcements.md [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
-* Fix Spyder installer not being able to finish installation due to Start Menu entry error in some Conda installations.
-* Fix Spyder installer not installing the right Spyder version (`6.0.0` vs `6.0.0rc2`)
-* Fix Binder instance with example workshop project from being non-responsive.
-* Fix errors related to unmaximazing panes and layout changes.
+* Several fixes and improvements to the update detection mechanism.
+* Fix SSH tunneling info handling for remote kernels connection and add remote client tests.
+* Fix issue related with Matplotlib inline backend registration.
+* Handle kernel fault file not being available.
+* Fix Debugger toolbar buttons and restore missing debugging buttons.
+* Update QtConsole constraint to 5.6.1 to support move cursor ANSI codes.
```
Comment (ccordoba12) [inline Announcements.md:20]:
```suggestion
* Fix plots not being generated with the Matplotlib inline backend.
* Restore missing debugger buttons to the main toolbar.
* Several fixes and improvements to the update detection mechanism.
* Fix SSH tunneling info handling for remote kernels connection and add remote client tests.
* Handle kernel fault file not being available.
* Update QtConsole constraint to 5.6.1 to support ANSI codes that move cursor.
```

**Review by ccordoba12 — COMMENTED** [review]
Some suggestions to reorganize and improve wording for changelog items.

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz!

**Comment by dalthviz** [comment]
@meeseeksdev please backport to 6.x

**Pull request merged**